user/tests/wiring/no_fixture/cellular.cpp

```diff
@@ -169,7 +168,16 @@ test(MDM_01_socket_writes_with_length_more_than_1023_work_correctly) {
 #endif // HAL_PLATFORM_NCP
 
     // https://github.com/spark/firmware/issues/1104
-    const char request[] =
+
+    const int dataSize = 1024;
+    const int bufferSize = 2048;
+    auto randData = std::make_unique<char[]>(1024);
```

Comment:
Use dataSize to initialize instead of magic number.

Reply:
Fixed.